docs/source/install/upgrades.rst

```diff
@@ -193,6 +193,12 @@ v3.5
      sudo rpm -e --nodeps nodejs
      sudo yum upgrade st2chatops
 
+* Redis server is installed and configured as backend for the coordination service
+  by default in the single node installation script to support workflows with multiple
+  branches and tasks with items. Upgrade requires coordination service to be setup
+  manually, For workflows to be executed properly, setup the coordination service
+  accordingly. See :doc:`../reference/policies` for setup instruction.
```

Comment:
The policies page doesn't really give instructions on how to install redis, just how to configure ST2 to point to it. So I don't know if we need to just put in here that you need to install a coordinations service, and then go to policies to configure it.

and/or maybe tell them that the manual installation instructions include steps to install redis which on a fresh install is now installed by default.

Reply:
I separated it out to a coordination section in below commit.